org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/tracking/CombinedTrackable.java

```diff
@@ -37,6 +39,10 @@ public CombinedTrackable(Trackable serverIssue, Trackable currentIssue) {
     this.creationDate = serverIssue.getCreationDate();
     this.resolved = serverIssue.isResolved();
     this.assignee = serverIssue.getAssignee();
+    this.severity = serverIssue.getSeverity();
+    // TODO when we are able to get server side issue type
+    // this.type = serverIssue.getType();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)